tracer/test/Datadog.Trace.ClrProfiler.Managed.Tests/Datadog.Trace.ClrProfiler.Managed.Tests.csproj

```diff
@@ -26,6 +26,7 @@
     <PackageReference Include="log4net" Version="$(Log4NetVersion)" />
     <PackageReference Include="Newtonsoft.Json" Version="13.0.2" />
     <PackageReference Include="NLog" Version="$(NLogVersion)" Condition=" $(TargetFramework.StartsWith('net4'))" />
+    <PackageReference Include="RabbitMQ.Client" Version="7.1.1" />
```

Comment:
What's this?

Reply:
The tests I added to this project (tests that don't use a "sample" app and don't rely on IL-rewriting) reference `BasicProperties` directly. I thought this was fine since there's many more libraries we already already referencing above (`AWSSDK.*`, `Confluent.Kafka`, etc).

Is there a better way to do this?

Reply:
Ahhhhhhh gotcha, I didn't realize they were in this project my bad 😄